 Fix SystemUI FC due to Google Assistant

We could call just the onConfigurationChanged but there's no need
to do the "visible" boolean checks when we create the view the first
time

 AndroidRuntime( 8424): Shutting down VM
 AndroidRuntime( 8424): FATAL EXCEPTION: main
 AndroidRuntime( 8424): Process: com.android.systemui, PID: 8424
 AndroidRuntime( 8424): java.lang.NullPointerException: Attempt to invoke virtual method 
 'com.android.systemui.assist.AssistOrbView com.android.systemui.assist.AssistOrbContainer.getOrb()' 
 on a null object reference

diff --git a/packages/SystemUI/src/com/android/systemui/assist/AssistManager.java b/packages/SystemUI/src/com/android/systemui/assist/AssistManager.java
index 3435d1d..0932f19 100644
--- a/packages/SystemUI/src/com/android/systemui/assist/AssistManager.java
+++ b/packages/SystemUI/src/com/android/systemui/assist/AssistManager.java
@@ -109,15 +109,7 @@ public void onConfigurationChanged() {
             visible = mView.isShowing();
             mWindowManager.removeView(mView);
         }
-
-        mView = (AssistOrbContainer) LayoutInflater.from(mContext).inflate(
-                R.layout.assist_orb, null);
-        mView.setVisibility(View.GONE);
-        mView.setSystemUiVisibility(
-                View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN | View.SYSTEM_UI_FLAG_LAYOUT_STABLE
-                        | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION);
-        WindowManager.LayoutParams lp = getLayoutParams();
-        mWindowManager.addView(mView, lp);
+        createOrbView();
         if (visible) {
             mView.show(true /* show */, false /* animate */);
         }
@@ -170,10 +162,24 @@ public void hideAssist() {
     }
 
     private void showOrb(@NonNull ComponentName assistComponent, boolean isService) {
+        if (mView == null) {
+            createOrbView();
+        }
         maybeSwapSearchIcon(assistComponent, isService);
         mView.show(true /* show */, true /* animate */);
     }
 
+    private void createOrbView() {
+        mView = (AssistOrbContainer) LayoutInflater.from(mContext).inflate(
+                R.layout.assist_orb, null);
+        mView.setVisibility(View.GONE);
+        mView.setSystemUiVisibility(
+                View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN | View.SYSTEM_UI_FLAG_LAYOUT_STABLE
+                        | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION);
+        WindowManager.LayoutParams lp = getLayoutParams();
+        mWindowManager.addView(mView, lp);
+    }
+
     private void startAssistInternal(Bundle args, @NonNull ComponentName assistComponent,
             boolean isService) {
         if (isService) {
